pyclaragenomics/claragenomics/bindings/graph.pxd

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from libcpp.pair cimport pair
+from libcpp.memory cimport unique_ptr
```

Comment:
Remove the unused cimports: unique_ptr, int8_t, int16_t, uint16_t

Reply:
Done

Reply:
And good catch! I forgot that our flake8 doesn't catch cython formatting issues